final_task/rss_reader/classes/news_class.py

```diff
@@ -0,0 +1,32 @@
+class News:
+    """Class that represent single entry of news
+
+    Uses for storing title, date, link, text and links from text
+    as an attributes of class.
+
+    Methods
+    -------
+    print_feed_title(self) - print to stdout title of rss feed.
+
+    print_news(self) - print to stdout title, date, link, text
+        and links from text in particular sequence.
+    """
```

Comment:
If you have docstrings in corresponding methods it is better to not have these descriptions in class docstring.
But you can move this to your documentation

Reply:
Fixed